packages/traveling-fastlane/Rakefile

```diff
@@ -115,6 +116,10 @@ def create_package target
   end
   sh "mkdir #{package_dir}/lib/ruby"
   sh "tar -xzf packaging/traveling-ruby-#{TRAVELING_RUBY_VERSION}-#{target}.tar.gz -C #{package_dir}/lib/ruby"
+
+  # Hack to replace traveling ruby's expired ca-bundle.crt with cURL's most recent one
+  sh "wget #{CA_CERT_URL} -O #{package_dir}/lib/ruby/lib/ca-bundle.crt"
```

Comment:
Since we rely on curl elsewhere in this file, I'd prefer to use curl here unless there's some bootstrapping issue (what copy of curl/wget runs here?)

Reply:
the person who is releasing traveling-fastlane will have their local copy of `curl`/`wget` run here. I used wget before just because it downloads recursively (see [here](https://unix.stackexchange.com/questions/47434/what-is-the-difference-between-curl-and-wget))

Reply:
the person who is releasing traveling-fastlane will have their local copy of `curl`/`wget` run here. I used wget before just because it downloads recursively (see [here](https://unix.stackexchange.com/questions/47434/what-is-the-difference-between-curl-and-wget))